gradle/artifact-settings.gradle

```diff
@@ -10,6 +10,7 @@ ext {
     mapboxArtifactScmUrl = 'scm:git@github.com:mapbox/mapbox-maps-android.git'
     mapboxArtifactLicenseName = 'BSD'
     mapboxArtifactLicenseUrl = 'https://opensource.org/licenses/BSD-2-Clause'
-    versionName = project.property('VERSION_NAME')
+    versionName = project.hasProperty('MODULE_VERSION_NAME') ? project.property('MODULE_VERSION_NAME') : project.property('VERSION_NAME')
```

Comment:
Mention flag in the DEVELOPING.md about the new option?

Reply:
Added a section in the DEVELOPING.md